autogpt_platform/backend/backend/blocks/ayrshare/_api.py

```diff
@@ -0,0 +1,482 @@
+from __future__ import annotations
+
+import json
+import logging
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Dict, List, Optional, Union
+
+from backend.util.request import Requests
+from backend.util.settings import Settings
+
+logger = logging.getLogger(__name__)
+
+settings = Settings()
+
+
+class AyrshareAPIException(Exception):
+    def __init__(self, message: str, status_code: int):
+        super().__init__(message)
+        self.status_code = status_code
+
+
+class SocialPlatform(str, Enum):
+    BLUESKY = "bluesky"
+    FACEBOOK = "facebook"
+    TWITTER = "twitter"
+    LINKEDIN = "linkedin"
+    INSTAGRAM = "instagram"
+    YOUTUBE = "youtube"
+    REDDIT = "reddit"
+    TELEGRAM = "telegram"
+    GMB = "gmb"
```

Comment:
wtf is gmb

Reply:
Google My Business apparently
```suggestion
    GOOGLE_MY_BUSINESS = "gmb"
```